databend-client/src/main/java/com/databend/client/DatabendClient.java

```diff
@@ -35,7 +35,7 @@ public interface DatabendClient extends Closeable {
 
     // get the next page of the query result
     // @return true if there is next page or no more result
-    boolean next();
+    boolean advance();
```

Comment:
Maybe `next` is more reasonable? 🤔

Reply:
`next()` should return a item.

Reply:
> `next()` should return a it

This func means return true if there is next page, so `next()` means has next is true.

Reply:
> > `next()` should return a it
> 
> This func means return true if there is next page, so `next()` means has next is true.

correct me if I am wrong， my understanding is：

- next : return next item, internally move or not
- hasNext:
- advance:  internally  move